File: PHP/uberBlack.php
<?php

require_once('car.php');

class UberBlack extends Car{
    public $typeCarAccepted;
    public $seatMaterials;

    public function __construct($license, $driver, $typeCarAccepted, $seatMaterials){
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatMaterials = $seatMaterials;
    }
}

?>

File: Python/UberBlack.py
from car import car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterials = []
    
    def __init__(self, license, driver, typeCarAccepted, seatsMaterials):
        super(UberBlack, self).__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterials = seatsMaterials